apps/sms/services/README.md

```diff
@@ -0,0 +1,114 @@
+# The layers in the Messages application
+
+## Terms explanation
+
+We'll define different layers whose interfaces managed by the Bridge (aka [threads.js](https://github.com/gaia-components/threads/)):
+
+* Controllers are really tied to a specific view. They'll encapsulate
```

Comment:
This is not set in stone yet, we'll see.

I just think there will be cases where such view-tied worker can be useful. But maybe we won't need it in the end.